core/src/main/resources/openapi-specs/eth2/signing/schemas.yaml

```diff
@@ -135,7 +135,20 @@ components:
               $ref: "#/components/schemas/ContributionAndProof"
           required:
             - contribution_and_proof
-
+    ValidatorRegistrationSigning:
+      allOf:
+        - type: object
+          properties:
+            type:
+              type: "string"
+            signingRoot:
+              description: 'signing root for optional verification if field present'
+              type: "string"
+            validator_registration_request:
```

Comment:
Don't think we need this additional level of nesting in the request data. For other requests we have just treat the top level as the message. So would have expected to have epoch and signing root at the top level without the additional intermediary validator_registration_request object.